pkg/network/events/monitor.go

```diff
@@ -15,24 +15,28 @@ import (
 	"time"
 
 	"go.uber.org/atomic"
-	"go4.org/intern"
 
 	"github.com/DataDog/datadog-agent/pkg/security/events"
 	sprobe "github.com/DataDog/datadog-agent/pkg/security/probe"
 	"github.com/DataDog/datadog-agent/pkg/security/secl/model"
 	"github.com/DataDog/datadog-agent/pkg/security/secl/rules"
+	"github.com/DataDog/datadog-agent/pkg/util/intern"
 	"github.com/DataDog/datadog-agent/pkg/util/log"
 )
 
 var theMonitor atomic.Value
 var once sync.Once
 var initErr error
 
+var si = intern.NewStringInterner()
```

Comment:
This is the reason I wanted to use the intern package instead. Its easy to miss having more than one interner object, which means you will still have string dupes in your program.

Reply:
Can we not do this as part of this PR? This does not look related to any of the ring buffer changes.

Reply:
I reverted all of these commits and started a thread in `#system-probe`